interface-definitions/system-event-handler.xml.in

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interfaceDefinition>
+  <node name="system">
+    <children>
+      <node name="event-handler" owner="${vyos_conf_scripts_dir}/system_event_handler.py">
```

Comment:
I'm pretty sure people will want to watch for multiple patterns very soon. I think the simplest way to allow that is to convert it to a tag node and make the systemd service multi-instance. The user can add `set system event-handler MyPattern pattern ...` and script for it, and it will create `vyos-event-handler@MyPattern`.

I believe RAM usage of simple Python processes is acceptable to make that approach viable. The advantage is that the daemon will not need any internal watch&restart functionality, systemd will take care of it.

If memory overhead is too much, then, of course, we may need to make it multi-threaded and capable of restarting dead threads.

Reply:
`tagNode` done!